chain-vote/src/gang/p256k1.rs

```diff
@@ -53,6 +60,7 @@ impl Coordinate {
 impl GroupElement {
     /// Size of the byte representation of `GroupElement`.
     pub const BYTES_LEN: usize = 65;
+    pub const HASH_MAP_LEN: usize = FieldElement::SIZE_BYTES;
```

Comment:
I think this should be removed as well, we should not require a struct to commit to a specific representation in a hashmap, also because the current implementation seems tied to a specific optimization